recipes/cpython/all/conanfile.py

```diff
@@ -53,286 +56,294 @@ class CPythonConan(ConanFile):
         "with_sqlite3": True,
         "with_tkinter": True,
         "with_curses": True,
-
-        # Python 2 options
-        "unicode": "ucs2",
-        "with_bsddb": False,  # True,  # FIXME: libdb package missing (#5309/#5392)
-        # Python 3 options
         "with_lzma": True,
 
         # options that don't change package id
         "env_vars": True,
     }
-
-    _autotools = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _version_number_only(self):
-        return re.match(r"^([0-9.]+)", self.version).group(1)
-
-    @property
-    def _version_tuple(self):
-        return tuple(self._version_number_only.split("."))
+    short_paths = True
```

Comment:
Python packages compiled versions of all of the modules shipped in the stdlib in a `__pycache__` folder. These paths end up being very long, you can look at some of the hooks in the earlier runs of the PR to see such file names (`./lib/python3.10/site-packages/setuptools/_distutils/__pycache__/versionpredicate.cpython-310.pyc`, for example).

It would be a bad idea to remove these, as I believe they would just get regenerated when they are used, breaking cache immutability.